components/landing/QuickStart/StartLocally.vue

```diff
@@ -0,0 +1,187 @@
+          <cv-button
+            v-for="option in optionsBlock.elements"
+            :key="option.label"
+            :title="option.label"
+            class="start-locally__option"
+            :class="{ 'start-locally__option_active': option.active }"
+          >
+            {{ option.label }}
+          </cv-button>
+        </cv-button-set>
+      </div>
+      <div>
+        <h4 class="start-locally__option-title">Terminal</h4>
+        <SyntaxHighlight
+          :label="title"
+          :code="codeToInstallQiskit"
+          lang="console"
+          btn-on-bottom
+          white-design
+        />
+      </div>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import SyntaxHighlight from '~/components/ui/SyntaxHighlight.vue'
+import AppLink from '~/components/ui/AppLink.vue'
+import { DOWNLOAD_ANACONDA_LINK } from '~/constants/appLinks'
+
+@Component({
+  components: { SyntaxHighlight, AppLink },
+  head () {
+    return {
+      link: [
+        { rel: 'stylesheet', href: 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/styles/atom-one-dark.min.css' }
+      ]
+    }
+  }
+})
+export default class extends Vue {
+  downloadAnaconda = DOWNLOAD_ANACONDA_LINK
+  installOptions = [
+    {
+      title: 'Languages',
+      alias: 'languages',
+      elements: [{
+        label: 'Python 3.5+',
+        active: true
+      }]
+    },
+    {
+      title: 'Qiskit Install',
+      alias: 'qiskit-install',
+      elements: [
+        {
+          label: 'Stable (Recommended)',
+          active: true
+        },
+        {
+          label: 'Source',
+          active: false
+        }
+      ]
+    },
+    {
+      title: 'Operating System',
+      alias: 'os',
+      elements: [
+        {
+          label: 'Linux',
+          active: false
+        },
+        {
+          label: 'Mac',
+          active: true
+        },
+        {
+          label: 'Windows',
+          active: false
+        }
+      ]
+    }
+  ]
+
+  title = 'Qiskit Install'
+  codeToInstallQiskit = `conda install qiskit macos -m qiskit
+
+# MacOS Binaries dont support CUDA, install from source if CUDA is needed`
+}
+</script>
+
+<style lang="scss" scoped>
+@import '~carbon-components/scss/globals/scss/typography';
+
+.start-locally {
+  width: 63%;
+  padding-top: $spacing-07;
+  padding-right: $spacing-07;
+
+  &__title {
+    @include type-style('productive-heading-02');
+    color: $cool-gray-80;
+    margin-bottom: $spacing-05;
+  }
+
+  &__description {
+    @include type-style('body-long-01');
+    max-width: 7 * $column-size-large;
+    margin-bottom: $spacing-07;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $purple-70;;
+  }
+
+  &__options {
+    display: flex;
+    flex-direction: column;
+    row-gap: $spacing-07;
+  }
+
+  &__options-group {
+    display: grid;
+    column-gap: $spacing-07;
+
+    &_languages {
```

Comment:
Instead of creating a modifier per section, make the container to have the following `grid-template-columns`:

```scss
&__options-group {
  grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
}
```

You can find more info about `auto-fit` at https://css-tricks.com/auto-sizing-columns-css-grid-auto-fill-vs-auto-fit/